XamlControlsGallery/Package.appxmanifest

```diff
@@ -41,5 +41,6 @@
   </Applications>
   <Capabilities>
     <Capability Name="internetClient" />
+    <uap6:Capability Name="graphicsCapture"/>
```

Comment:
Why did we only update the non debug package file? Is it because the debug version will never be published to the store?

Reply:
I didn't realize there was another one! I added the capability there as well.